index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Green-on-Mind Design Designing Sustainably</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+        href="https://fonts.googleapis.com/css2?family=PT+Sans:wght@400;700&family=Sansita+Swashed:wght@600;800&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <nav class="site-nav">
+            <button type="button" class="u-button-reset js-toggle-trigger" aria-controls="site-menu">
+                <span aria-hidden="true" class="menu-icon">
+                    <div class="menu-icon__line"></div>
+                    <div class="menu-icon__line"></div>
+                    <div class="menu-icon__line"></div>
+                </span>
+            </button>
+            <ul id="site-menu" class="u-list-reset menu-list">
+                <li><a href="#details">details</a></li>
+                <li><a href="#speakers">speakers</a></li>
+                <li><a href="#register">register</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <h1>Green-on-Mind Design</h1>
+    <main>
+        <section class="details u-list-reset" id="details">
```

Comment:
This shouldn't need the `u-list-reset` class since that's for removing the default styles on a `ul` or `ol`.
```suggestion
        <section class="details" id="details">
```